.github/workflows/test-stellar.yaml

```
@@ -74,3 +89,117 @@ jobs:
 
       - name: Deploy AxelarGateway
         run: node stellar/deploy-contract deploy AxelarGateway --version ${{ steps.commit_hash.outputs.hash }}
+
+      - name: Deploy InterchainTokenService
+        run: node stellar/deploy-contract deploy InterchainTokenService --version ${{ steps.commit_hash.outputs.hash }}
+
+      - name: Deploy AxelarExample
+        run: node stellar/deploy-contract deploy AxelarExample --version ${{ steps.commit_hash.outputs.hash }}
+
+      - name: Deploy Upgrader
+        run: node stellar/deploy-contract deploy Upgrader --version ${{ steps.commit_hash.outputs.hash }}
+
+      - name: Deploy Multicall
+        run: node stellar/deploy-contract deploy Multicall --version ${{ steps.commit_hash.outputs.hash }}
+
+      ###### Command: Gateway ######
```

```suggestion
      ###### Command: AxelarGateway ######
```
use the same name as the contract